as/src/base/thr_demarshal.c

```diff
@@ -436,7 +436,7 @@ thr_demarshal(void *unused)
 
 		// Iterate over all events.
 		for (i = 0; i < nevents; i++) {
-			cf_socket *ssock = events[i].data;
+			cf_socket *ssock = events[i].data.ptr;
```

Comment:
commit message could say we're using the standard provided union of auxiliary data as per the epoll_ctl man page.

Reply:
done.